pkg/plugins/pluginscdn/pluginscdn.go

```diff
@@ -3,15 +3,14 @@ package pluginscdn
 import (
 	"errors"
 	"fmt"
-	"net/url"
-	"path"
+	"strings"
 
 	"github.com/grafana/grafana/pkg/plugins/config"
 )
 
 const (
-	// systemJSCDNKeyword is the path prefix used by system.js to identify the plugins CDN.
-	systemJSCDNKeyword = "plugin-cdn"
+	// systemJSCDNURLTemplate is a special path template used by system.js to identify plugin CDN assets
+	systemJSCDNURLTemplate = "plugin-cdn/{id}/{version}/public/plugins/{id}/{assetPath}"
```

Comment:
Considering this is baked into the code this way, should the plugin CDN URL config value maybe instead be just the domain instead? 🤔

Reply:
Yeah I think it makes more sense and it'll be consistent. I'll change that! :+1: